datahub-web-react/src/app/entityV2/shared/tabs/Dataset/Queries/QueryModal.tsx

```diff
@@ -76,20 +75,21 @@ type Props = {
 export default function QueryModal({ query, title, description, showDetails = true, onClose }: Props) {
     return (
         <StyledModal
-            visible
+            open
             width={MODAL_WIDTH}
-            title={null}
+            title="Query"
             closable={false}
-            onCancel={onClose}
+            onCancel={() => onClose?.()}
```

Comment:
not in this scope i don't think but curious why `onClose` here is optional - we wouldn't want to get into a situation where this function isn't defined and the modal never closes